### DISCUSSION TITLE
Font not working

### DISCUSSION THREAD

**Discussion by kevinpcf** [tailwindlabs/tailwindcss#10704] [Help] [answered]
I'm working on a React js app with Tailwindcss, all Tailwind properties works but the font does not be updated when I run app. If I put font with style prop, instaed of className the font is showed correctly.

There is my page:

```
<div className="flex flex-row mr-8">
   <p className="text-white font-bold font-body">
    SALE ESPORTS
   </p>
</div>
```
This is my tailwind.config.js file

```
module.exports = {
  content: ["./src/**/*.{html, js, ts, jsx, tsx}"],
  theme: {
    fontFamily: {
      body: ["Chakra Petch"],
      gaming: ["Press Start 2P"],
      icons: ["Material Icons"],
    },
    extend: {
      colors: {
        primary: {
          accent: "#30ACFF",
          main: "#4448BB",
          dark: "#000A3B",
        },
        secondary: {
          accent: "#E84AE6",
          main: "#C01EE1",
          dark: "#70288F",
        },
        backColor: "#1A1B1F",
      },
    },
  },
  plugins: [],
};
```
There's my index.css

```
@tailwind base;
@tailwind components;
@tailwind utilities;
```
And in my index.html page I get the link from google fonts:

```
<link
      href="https://fonts.googleapis.com/css2?family=Chakra+Petch&display=swap"
      rel="stylesheet"
    />
<link rel="preconnect" href="https://fonts.googleapis.com" />
<link href="/dist/output.css" rel="stylesheet" />
```
In this picture I show the project structure: https://i.stack.imgur.com/0Y1ZR.png

**Comment by wongjn** [comment, marked as answer]
You might need to quote the font family value:
```diff
 fontFamily: {
-  body: ["Chakra Petch"],
+  body: ["'Chakra Petch'"],
   gaming: ["Press Start 2P"],
```

**Reply by kevinpcf** [reply]
I tried but it still doesn't work

**Reply by wongjn** [reply]
Consider providing a remotely-hosted git repository that reproduces the unexpected behavior.

**Reply by kevinpcf** [reply]
you can check on this public repo: https://github.com/kevinpcf/react-js-app

**Reply by wongjn** [reply]
Thank you for providing that. There were some other misconfigurations I spotted. Once these were resolved, the text appeared with the custom font.

## `content` glob malformed

The multiple extensions within the braces should not be space separated:

`tailwind.config.js`

```diff
 module.exports = {
-  content: ["./src/**/*.{html, js, ts, jsx, tsx}"],
+  content: ["./src/**/*.{html,js,ts,jsx,tsx}"],
   theme: {
```

## `import` the stylesheet

The `assets/index.css` is not `import`ed in the main app code anywhere and thus it is never loaded.

`src/index.tsx`

```diff
 import React from "react";
 import ReactDOM from "react-dom";
+import './assets/index.css';

 ReactDOM.render(
```

**Reply by kevinpcf** [reply]
It works now!!! Thank you so much
